pkg/controller/plan/kubevirt.go

```diff
@@ -219,6 +219,37 @@ func (r *KubeVirt) GetImporterPod(pvc core.PersistentVolumeClaim) (pod *core.Pod
 	return
 }
 
+// Get the importer pods for a PersistentVolumeClaim.
+func (r *KubeVirt) GetImporterPods(pvc core.PersistentVolumeClaim) (pods []core.Pod, found bool, err error) {
+	pods = []core.Pod{}
+	if pvc.Annotations[AnnImporterPodName] == "" {
+		return
+	}
+
+	podList := &core.PodList{}
+
+	err = r.Destination.Client.List(
+		context.TODO(),
+		podList,
+		&client.ListOptions{
+			Namespace:     r.Plan.Spec.TargetNamespace,
+			LabelSelector: labels.SelectorFromSet(map[string]string{"app": "containerized-data-importer"}),
+		},
+	)
+	if err != nil {
+		err = liberr.Wrap(err)
+		return
+	}
+	for _, pod := range podList.Items {
+		if strings.Contains(pod.Name, fmt.Sprintf("importer-%s-%s", r.Plan.Name, pvc.Annotations[kVM])) {
```

Comment:
can we propagate labels to the pod via the DV? if so, it will not help now, but perhaps in the future

Reply:
we discussed it here:
https://github.com/kubev2v/forklift/pull/588#discussion_r1318701734
and i opened issue for CDI for this feature.